front/src/common/Map/Search/MapSearchOperationalPoint.tsx

```diff
@@ -178,7 +178,7 @@ const MapSearchOperationalPoint = ({
                 {searchResult.name}
                 <span className="ch">{searchResult.ch}</span>
               </span>
-              <span className="uic">{searchResult.uic}</span>
+              <span className="uic">{searchResult.ci}</span>
```

Comment:
shouldn't you change the class uic for ci ?

Reply:
That's what we want: display only ci (as explained) BUT search in UIC.